src/app/assinante/service/assinante.service.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Router } from  '@angular/router';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AssinanteService {
+  public artPlusURL = `${environment.apiURL}`;
+  public token = localStorage.getItem('token');
+
+  httpOptions = {
+    headers: new HttpHeaders({
+      'Content-Type': 'application/json',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8399665)
```suggestion
      "Content-Type": 'application/json',
```